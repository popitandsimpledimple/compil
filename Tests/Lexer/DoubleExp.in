123e123
%10101.e9
$ABCD.e9
&777.e9
9.e+1
9.1e1
9.e1
$F.e+1
$F.e1
$F.1e1
%1.e+1
%1.1e1
%1.e1
&7.e+1
&7.1e1
&7.e1