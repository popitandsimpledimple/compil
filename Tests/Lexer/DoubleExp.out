1	1	Double	1.2300000000000001E+125	123e123
2	1	Double	2.1000000000000000E+010	%10101.e9
3	1	Double	4.3981000000000000E+013	$ABCD.e9
4	1	Double	5.1100000000000000E+011	&777.e9
5	1	Double	9.0000000000000000E+001	9.e+1
6	1	Double	9.1000000000000000E+001	9.1e1
7	1	Double	9.0000000000000000E+001	9.e1
8	1	Double	1.5000000000000000E+002	$F.e+1
9	1	Double	1.5000000000000000E+002	$F.e1
10	1	Double	1.5100000000000000E+002	$F.1e1
11	1	Double	1.0000000000000000E+001	%1.e+1
12	1	Double	1.1000000000000000E+001	%1.1e1
13	1	Double	1.0000000000000000E+001	%1.e1
14	1	Double	7.0000000000000000E+001	&7.e+1
15	1	Double	7.1000000000000000E+001	&7.1e1
16	1	Double	7.0000000000000000E+001	&7.e1
